widget_driver_annotation/lib/src/driver_annotations.dart

```diff
@@ -86,3 +86,72 @@ class TestDriverDefaultFutureValue<T> {
   final T? value;
   const TestDriverDefaultFutureValue([this.value]);
 }
+
+class _DriverProvidableProperty {
+  const _DriverProvidableProperty();
+}
+
+/// Use this annotation on constructor params, which should be able to be passed from the `DrivableWidget` to the `
```

Comment:
should we maybe move this to readme/example.md file?

Reply:
We could have both, documentation in code and in README